src/main/TreeCapitator.java

```diff
@@ -481,10 +507,11 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 							accentColor + "Invincible replant: " + textColor
 									+ (invincibleReplant ? "enabled" : "disabled"),
 							accentColor + "Axe Needed: " + textColor + (axeNeeded ? "yes" : "no"),
+                            accentColor + "Axe Type Required: " + textColor + (AxeTypeInput.isValid(axeTypeRequired) ? axeTypeRequired : "disabled")
 							accentColor + "Axe Damaged: " + textColor + (axeNeeded ? "yes" : "no"),
 							accentColor + "Damage Axe: " + textColor + (damageAxe ? "yes" : "no"),
-							accentColor + "Break Axe: " + textColor + (breakAxe ? "yes" : "no"), 
-							accentColor + "Ignore Leaves: " + textColor + (ignoreLeaves ? "yes" : "no"), 
+							accentColor + "Break Axe: " + textColor + (breakAxe ? "yes" : "no"),
+							accentColor + "Ignore Leaves: " + textColor + (ignoreLeaves ? "yes" : "no"),

```

Comment:
No actual changes on L513 and L514. Looks like my editor removed some trailing whitespace.